jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/handler/ContextHandlerTest.java

```diff
@@ -1061,4 +1062,54 @@ public boolean handle(Request request, Response response, Callback callback) thr
         assertTrue(shutdown.isDone());
         assertThat(gracefulHandler.getCurrentRequestCount(), is(0L));
     }
+
+    @Test
+    public void testContextDump() throws Exception
+    {
+        Server server = new Server();
+        ContextHandler contextHandler = new ContextHandler("/ctx");
+        server.setHandler(contextHandler);
+        contextHandler.setHandler(new Handler.Abstract()
+        {
+            @Override
+            public boolean handle(Request request, Response response, Callback callback) throws Exception
+            {
+                callback.succeeded();
+                return true;
+            }
+
+            @Override
+            public String toString()
+            {
+                return "TestHandler";
```

Comment:
This name isn't appearing in the dump. Should it be part of the dump or was it forgotten?

Reply:
line 3:
```
oejsh.ContextHandler@3b0fe47a{/ctx,/ctx,b=null,a=AVAILABLE,h=TestHandler} - STARTED
+= SymlinkAllowedResourceAliasChecker@1750fbeb{base=null,protected=[]} - STARTED
+= TestHandler - STARTED
+- org.eclipse.jetty.server.AllowedResourceAliasChecker$AllowedResourceAliasCheckListener@5b247367
+> No ClassLoader
+> handler attributes size=3
```

Reply:
Where did I have my eyes?